packages/contracts/hardhat.config.js

```diff
@@ -0,0 +1,204 @@
+require("@nomiclabs/hardhat-waffle");
```

Comment:
It seems like this file is coming back; I thought we rebuilt it purposely with less complexity.